templates/repo/home.tmpl

```diff
@@ -8,7 +8,7 @@
 		<div class="ui repo-description">
 			<div id="repo-desc">
 				{{$description := .Repository.DescriptionHTML $.Context}}
-				{{if $description}}<span class="description">{{$description}}</span>{{else if .IsRepositoryAdmin}}<span class="no-description text-italic">{{.locale.Tr "repo.no_desc"}}</span>{{end}}
+				{{if $description}}<span class="description">{{$description | RenderCodeBlock}}</span>{{else if .IsRepositoryAdmin}}<span class="no-description text-italic">{{.locale.Tr "repo.no_desc"}}</span>{{end}}
```

Comment:
Surprising that that passes the validation…
I would expect that to escape the HTML tags as it is (obviously) unsafe…

Reply:
You are right, it's possible to inject HTML here, wonder if that was the case before.

Reply:
Can confirm HTML injection was possible before. It's sanitized but imho we should remove this.